src/tabs/__integ__/in-memory.test.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+
+import createWrapper from '../../../lib/components/test-utils/selectors';
+
+const wrapper = createWrapper();
+
+const setupTest = (testFn: (page: BasePageObject) => Promise<void>) => {
+  return useBrowser(async browser => {
+    const page = new BasePageObject(browser);
+    await browser.url('#/light/tabs/in-memory-integ');
+    await page.waitForVisible(wrapper.findTabs().findTabContent().toSelector());
+    await testFn(page);
+  });
+};
+
+test(
+  'eager-loaded tab content should already be available',
+  setupTest(async page => {
+    await page.pause(2000);
```

Comment:
The dependency on page timeout can be easily removed here, I think. For instance, when the timeout inside the component is resolved you can set some flag to the global object or add a DOM node to the bottom of the page, e.g. `<div id="loaded">Tab loaded</div>`. Then you can query it in the test using waitForAssertion() helper.